src/plugin/nodeSpec.ts

```diff
@@ -161,7 +161,7 @@ export const getNodeSpecForField = (
           parseDOM: getDefaultParseDOMForLeafNode(elementName, fieldName),
           attrs: {
             fields: {
-              default: { value: field.defaultValue },
+              default: field.defaultValue,
```

Comment:
Another potentially contentious change here. This PR changes how default custom field attributes are modelled. This is due to how newNodes are created and how `setNodeMarkup` treats `undefined` as a missing argument. The default needs to be undefined, otherwise we can't set the attribute value to it.

Not the best explanation, happy to go into more detail if that's helpful.

Reply:
That does make sense, and we should support `undefined` as a possible attribute value – 👍